MVPInternMarsCompetition/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,288 @@
+﻿using MVPInternMarsCompetition.Data;
+using MVPInternMarsCompetition.Pages;
+using MVPInternMarsCompetition.Utilities;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace MVPInternMarsCompetition.Tests
+{
+    [TestFixture]
+    public class CertificationTest : BaseClass
+    {
+        CertificationPage certificationPageObj = new CertificationPage();
+
```

Comment:
you are missing the json files and json reader which should be used to pass the test data please add this